controllers/volsync/vshandler_test.go

```diff
@@ -664,6 +664,14 @@ var _ = Describe("VolSync_Handler", func() {
 					Expect(k8sClient.Create(ctx, dummyPSKSecret)).To(Succeed())
 					Expect(dummyPSKSecret.GetName()).NotTo(BeEmpty())
 
+					// Make sure the secret is created to avoid any timing issues
```

Comment:
This addition of eventually is good and is the right fix. 

@BenamarMk What is the use of GetName() after Create().